model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -344,6 +343,11 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
     # Create a new copy of graph with correct attributes (shape & type infer, backend attrs etc.)
     for op in graph.get_op_nodes():
 
+        # Save input shapes restored from IR
+        op['old_input_shapes'] = list()
+        for n in op.in_nodes():
+            op.old_input_shapes.append(int64_array(op.in_node(n).shape))
```

Comment:
please use new Graph API here.

Reply:
We use this code for restoring graph and new Graph API using here not always possible.
Also this change may have unclear impact to existed pipelines.